code/app/be/objectify/deadbolt/java/AbstractDeadboltHandler.java

```diff
@@ -37,6 +38,9 @@
 {
     protected final DeadboltExecutionContextProvider executionContextProvider;
 
+    private static final AtomicLong NEXT_ID = new AtomicLong(0);
+    private final long id = NEXT_ID.getAndIncrement();

```

Comment:
BTW: Instead of `long` we could also use `UUID` for the id and use `UUID.randomUUID()`. However I don't think this is really relevant.